### DISCUSSION TITLE
Running docker-redis-cluster on Windows Server?

### DISCUSSION THREAD

**Discussion by mlevinmail** [Grokzen/docker-redis-cluster#118] [General]
Would this repo work on Windows Server?

**Comment by Grokzen** [comment]
@mlevinmail I dont know, i do not use docker on windows. If they impelment everything the same thereis nothing inside the container at least that should prevent it. If there is any issues that is due to the docker implementation itself on windows and nothing that this image do.

Please try tho and report back your findings.

**Reply by tangnotes** [reply]
This way should work:

```
docker run -e "IP=0.0.0.0" -p 7000-7005:7000-7005 grokzen/redis-cluster:latest
```
